lib/commands/find.js

```diff
@@ -23,12 +23,21 @@ helpers.findUIElementOrElements = async function  (strategy, selector, mult, con
   if (strategy !== "xpath") {
     selector = util.escapeSpecialChars(selector, "'");
   }
+
   if (typeof context === "undefined" || !context) {
     context = '';
   } else if (typeof context === "string") {
     context = util.escapeSpecialChars(context, "'");
   }
 
+  if (strategy === "xpath" && selector === "//*[@scrollable=true]") {
```

Comment:
it should be `@scrollable="true"`, otherwise this xpath expression does not comply with query format definition.